gravitee-am-model/src/main/java/io/gravitee/am/model/Application.java

```diff
@@ -172,6 +175,14 @@ public void setIdentities(Set<String> identities) {
         this.identities = identities;
     }
 
+    public Map<String, Integer> getOrderedIdentities() {
+        return orderedIdentities;
```

Comment:
I've mentioned this on other PRs but I think we should get into the habit of returning empty collections, rather than nulls. We have a lot of `null` checks on conditionals in our codebase that could and should be unnecessary clutter. In this instance, the field `orderedIdentities` would have a default value of `emptyMap()`.